lib/recurly/client.php

```diff
@@ -141,6 +149,14 @@ private function _sendRequest($method, $uri, $data = '')
       curl_setopt($ch, CURLOPT_CUSTOMREQUEST, $method);
     }
 
+    if (Recurly_Client::$logger) {
+      Recurly_Client::$logger->info('Send request to Recurly', array('method' => $method, 'uri' => $uri));
+      Recurly_Client::$logger->debug('Send request with headers', array('headers' => $headers));
+      if ($data) {
+        Recurly_Client::$logger->debug('Recurly request body', array('body' => $data));
+      }
+    }
```

Comment:
Ah I'd commented on an old version of this. I'll re-paste what I'm playing with:
```
    if (Recurly_Client::$logger) {
      Recurly_Client::$logger->info('Recurly Request', array('method' => $method, 'uri' => $uri));
      Recurly_Client::$logger->debug('Request data', array('headers' => $headers, 'body' => $data));
    }
```